src/components/Rocket.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/extensions */
+/* eslint-disable camelcase */
+import './Rockets.css';
+import { PropTypes } from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { rocketreserve } from '../redux/Rockets/Rocketslice';
+
+const Rocketer = ({ Rocket }) => {
+  const {
+    rocket_id,
+    rocket_name,
+    description,
+    flickr_images,
+    reserved,
+  } = Rocket;
+  const dispatch = useDispatch();
+  const handleclick = () => {
+    dispatch(rocketreserve(rocket_id));
+  };
+  return (
+    <div id={rocket_id} data-testid="rocketId" className="rockets">
+      <img src={flickr_images} alt="" />
+      <div className="rockets-children">
+        <h3>
+          { rocket_name }
+        </h3>
+        <p>
+          <span>{reserved ? 'reserved' : ''}</span>
+          { description }
+        </p>
+        <p>
+          {(reserved && (<button type="button" onClick={(e) => handleclick(e)} className="btn2">cancel reservation</button>))}
+          {(!reserved && (<button type="button" onClick={(e) => handleclick(e)} className="btn">reserve rocket</button>))}
+        </p>
+      </div>
+    </div>
+  );
+};
+
+Rocketer.propTypes = {
+  Rocket: PropTypes.shape({
+    rocket_id: PropTypes.string,
+    rocket_name: PropTypes.string,
+    description: PropTypes.string,
+    flickr_images: PropTypes.string,
+    reserved: PropTypes.string,
```

Comment:
- Optional
You are validating the `reserved` prop as a string but a boolean value is being supplied. I would recommend fixing this issue.
![image](https://user-images.githubusercontent.com/33268927/226121744-ce4bf88b-d0a8-40d5-b008-5bde0f056d0b.png)